server/code_switch/cs_level_identification.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import langid
+import json
+import codecs
+
+LANGUAGES = ['en', 'es']
+langid.set_languages(langs=LANGUAGES)
+
+MAJOR_LANGUAGE_CODES = {'en': 'E', 'es': 'S'}
+UNCOMMON_WORDS = json.load(open('resources/translation_pairs.json'))
+
+def get_other_lang(lang):
+	major_lang_index = LANGUAGES.index(lang)
+	return LANGUAGES[major_lang_index-1]
+
+def load_words(filename):
+	with codecs.open(filename, 'r', 'utf-8') as f:
+		txt_str = f.read()
+		list_of_words = txt_str.split()
+	f.close()
+	return list_of_words
+
+
+COMMON_WORDS = {'en': load_words('resources/most_common_en_words_without_shared.txt'),
+                  'es': load_words('resources/most_common_es_words_without_shared.txt')}
+
+
+def get_other_lang(lang):
+	major_lang_index = LANGUAGES.index(lang)
+	return LANGUAGES[major_lang_index-1]
+
+
+def check_phrasal_cs_presence(sentence, minor_lang):
+	tokens = sentence.split()
+	if len(tokens)<=2:
+		return False
+	else:
+		for n in range(len(tokens)-2):
+			triplet = ' '.join(tokens[n:n+3])
+			if langid.classify(triplet)[0] == minor_lang:
+				return True
+	return False
+
+def check_lexical_cs_presence(sentence, minor_lang):
+	tokens = sentence.split()
+	for token in tokens:
+		if token in UNCOMMON_WORDS[minor_lang]:
+			return True
+	return False
+
+def check_frozen_expression_cs_presence(sentence, minor_lang):
+	tokens = sentence.split()
+	for token in tokens:
+		if token in COMMON_WORDS[minor_lang]:
+			return True
+	return False
+
+
+def clean_sentence(sentence):
+	chars_to_remove = '?!,.:"\''
+	for char in chars_to_remove:
+		sentence = sentence.replace(char, '')
+	return sentence.strip()
+
+def calc_cs_level(sentence):
```

Comment:
consider making it a class with 1 public function, this one. and all others are private. also add docs / types

Reply:
I tried to fix the class as you suggested.